src/DatePicker/index.jsx

```diff
@@ -101,6 +101,7 @@ const DatePicker = forwardRef((props, ref) => {
     const min = props.min || Date.now();
 
     if (itemTimestamp > max) return false;
+    if (props.min === null) return true;
```

Comment:
`const min = props.min || Date.now()` ([line 101](https://github.com/sociomantic-tsunami/nessie-ui/pull/968/files#diff-a5548d0912da20895489432aa630e773R101)) could be moved down  after this check